deployment/deployment.json

```diff
@@ -112,7 +112,7 @@
         "status": "prod",
         "versions": {
           "schema": "2.0.1",
-          "subgraph": "1.2.13",
```

Comment:
I don't think we should bump the version before it gets deployed to prod

Reply:
or maybe we can have a `subgraph-dev` and a `subgraph-prod`, or just a `subgraph` and a `dev`